### PR TITLE
Sara development

### PR DESCRIPTION
**Pull request by kolbsara**
Week 1 complete

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,10 +8,13 @@
...
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Sara's Portfolio</h1>
+      <p>I love art and I love problem solving, i.e. coding! I'm passionate about film and acting, and my hope is to integrate technology with that passion in the future. I'm also deeply passionate about the enviornment and using technology as a means to empower people. I hope to pursue socially driven software projects</p>
+      <a href="https://github.com/kolbsara">Visit my github!</a>
+      <div></div>
```
Comment (sbahaud) [inline portfolio/src/main/webapp/index.html:14]: If the <div> is just for adding space? You could wrap the <a> tag in a div.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -2,8 +2,10 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  align-items: center;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css:5]: Keep the properties in alphabetical order.
aligh-items:
margin-left:
margin-right:
width:

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -2,8 +2,10 @@
...
 }
 
 #greeting-container {
   margin-top: 20px;
+  align-items: center;
```
Comment (sbahaud) [inline portfolio/src/main/webapp/style.css:10]: Same here. Make sure the properties within a rule are in alphabetical order.

**Comment by kolbsara** [comment]
Fixed them about to update changes

**Pull request closed without merging**